### ISSUE TITLE
Improvement: Auto Updater

### ISSUE THREAD

**Issue by dark0ni** [notepad-plus-plus/notepad-plus-plus#10338]
### Description of the Improvment
When the notepad++ gets an update it will ask you for a installation. Then you must step by step click through wizard to install the update.

It would be useful when the updater would automatic install the update to the right location. For example in the browser edge or the IDE intellij he shows a update icon instead popup. When you press install update and the rest is automated. 

This improvement would be great because notepad++ receive a lot of updates, which is great, but the installation progress could be more optimized.

P.S. Thank you for this great notepad, its for me a daily driver.

**Comment by nbourre** [comment]
Indeed, something less intrusive such as a bell in the right corner of the status bar. By popping up a window, it breaks the user _workflow_. 

The update notification could be launched after the user quits.

**Comment by CennoxX** [comment]
Duplicate of #8514
